sphinx/about/whats-new.rst

```diff
@@ -1,6 +1,64 @@
 Version history
 ===============
 
+6.2.1 (2019 August)
+-------------------
+
+File format fixes and improvements:
+
+* Applied Precision CellWorX
+   - corrected plane positions for series index > 0 rather than reusing positions from the first series
+
+* DeltaVision
+   - added objective info for new Applied Precision 100X/1.4 lensID
+   - updated so that date from dv file will override log file date to avoid locale dependent dates
+
+* Leica LAS AF LIF (Leica Image File Format)
+   - fixed units and indexing for tile-based plane positions
+
+* TIFF
+   - fixed a potential exception in MinimalTiffReader when the TIFF is stored using very 
+     large tile/strips dimensions
+
+* Zeiss CZI
+   - fixed a bug to prevent an index out of bounds exception when populating positions
```

Comment:
Odd to see "fixed a bug" not in the adjacent "bug fixes" section.

Reply:
Updated the wording slightly with the latest commit to prevent confusion